packages/firebase_admob/android/src/main/AndroidManifest.xml

```diff
@@ -1,9 +1,4 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="io.flutter.plugins.firebaseadmob">
-
-  <!-- Sample AdMob App ID: ca-app-pub-3940256099942544~3347511713 -->
-  <meta-data
```

Comment:
Is this intentional?

Reply:
Yes, I removed this because the user should be adding this to their app `AndroidManifest.xml` and not to the plugin code. It didn't cause any problems with compilation or tests when I removed it.